tests/test_migrator.py

```diff
@@ -1,9 +1,35 @@
-# -*- coding: utf-8 -*-
+# from mock import patch
 
-import unittest
```

Comment:
You should probably use `pytest` which is much more modern and flexible as a Python unit testing framework - even CKAN has moved to it in 2.9 after many years. 

You also don't need to put tests in classes unless there is a good reason (such as creating "inherited tests"), just writing test functions with some fixtures is much simpler and preferred. However, this is not a requirement, just a common practice.